Sources/AppStoreConnectCLI/Services/Options/ListUserOptions.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+
+struct ListUsersOptions {
+    let limitVisibleApps: Int?
+    let limitUsers: Int?
+    let sort: ListUsers.Sort?
+    let filterUsername: [String]
```

Comment:
I guess all of the filters can be optional when requesting?

Reply:
If there are no filters we can just pass in an empty array

Reply:
We could make it `[String]?` but before it was `[String?]` which wasn't very useful to us

Reply:
Turns out there's no Option initializer for `Optional<Array<ArugumentParsable>>` so `[String]` will have to do